--- libsvgtiny-0.1.0/Makefile~	2013-04-19 22:26:48.000000000 +0300
+++ libsvgtiny-0.1.0/Makefile	2013-04-27 17:53:19.066706553 +0300
@@ -45,7 +45,8 @@
 include $(NSBUILD)/Makefile.top
 
 # Extra installation rules
+lib := lib
 I := /include
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/svgtiny.h
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib/pkgconfig:lib$(COMPONENT).pc.in
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib:$(OUTPUT)
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib)/pkgconfig:lib$(COMPONENT).pc.in
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib):$(OUTPUT)
